soccer/gameplay/tactics/positions/submissive_defender.py

```diff
@@ -7,20 +7,25 @@
 from enum import Enum
 import math
 import planning_priority
+import evaluation
 
 
 ## Defender behavior meant to be coordinated in a defense tactic
 # The regular defender does a lot of calculations and figures out where it should be
 # This defender lets someone else (the Defense tactic) handle calculations and blocks things based on that
 class SubmissiveDefender(
-        single_robot_composite_behavior.SingleRobotCompositeBehavior):
+    single_robot_composite_behavior.SingleRobotCompositeBehavior):
     class State(Enum):
         ## gets between a particular opponent and the goal.  stays closer to the goal
         marking = 1
-        # TODO: add clear state to get and kick a free ball
+        clearing = 2
 
     def __init__(self):
         super().__init__(continuous=True)
+
+        self.go_clear = False
+        self.safety_multiplier = 0.9
```

Comment:
It would be good to add a comment for `safety_multiplier` specifying what it changes.  It'll make it more straightforward if we need to tune the value later.